thoughtis/cata-seo
# Issue#1/consolidate seo code

### Related Issues
- Closes #1 

### What Was Accomplished
- Consolidated common SEO-related code from our current themes & plugins into a single plugin

### How It Was Tested
- [x] Locally on parent theme
- [x] Locally on child theme
- [x] Child theme develop

### How To Test
- Structured data appears in the footer with this plugin activated
- Customizer is available on block themes and includes a share image field
- `noindex, follow` meta tag is present on attachments, get requests, paged content, search, and 404 pages
- Feeds are disallowed in robots.txt
- Breadcrumb structured data needs to be tested locally. Change [one of the filters in the Thought Catalog theme](https://github.com/wpcomvip/thought-catalog/blob/master/themes/thoughtcatalog-2014/includes/breadcrumb/class-breadcrumb.php#L30-L31) to the ~`cata_structured_data_breadcrumbs`~ `cata_breadcrumbs_get_crumbs` hook and ensure the structured data appears in the footer.

### Open Questions
- The Cata Breadcrumbs plugin will handle adding categories by default to the `cata_structured_data_breadcrumbs` filter hook.
